### PR TITLE
Added Robin Hodl

### PR DESCRIPTION
**Pull request by robinhodl69**

### CODE REVIEW

**Comment by robinhodl69** [comment]
https://polkadot.subscan.io/extrinsic/15960045-2

**Comment by stepanLav** [comment]
Hello @robinhodl69, 
Appreciate your request. However, I noticed that there are some conflicts with previous data. Would it be possible to resolve these conflicts?
Thank you!

File: registry/polkadot.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-        "longDescription": "## About us\nChaosDAO is a curated community of Polkadot & Kusama natives. Our members are involved with every aspect of the ecosystem as developers, community organizers, [Twitter](https://twitter.com/ChaosDAO) and YouTube educators, and as ordinary retail investors.\n\n## Voting Process\nAs part of our own organization’s evolution we have developed a robust [internal governance mechanism](https://github.com/ChaosDAO-org/ChaosDAO_docs/blob/main/governance/OpenGov.md) to determine how our community will publicly vote on referenda. By voting on behalf of the long-time Dotsama community we strive to ensure that all votes we cast reflect the true best interests of our native ecosystem.\n\n![ChaosDAO Manifesto image](https://raw.githubusercontent.com/ChaosDAO-org/ChaosDAO_docs/main/.gitbook/assets/ChaosDAO_Manifesto_600px.jpg)",
-        "isOrganization": true
+        "address": "15vvAhHcmop8PjP5QNr3mXzX1oesv6vjun25iSgMe9YpDezv",
+        "name": "RobinHodl",
+        "image": "https://github.com/robinhodl69/opengov-delegate-registry/blob/master/images/RobinHodl.png",
```
Comment (stepanLav) [inline registry/polkadot.json:5]: Also, url should be in that format:
```suggestion
        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/RobinHodl.png",
```

File: registry/kusama.json [hunk 2]
```diff
@@ -190,5 +190,13 @@
         "shortDescription": "CertHum.com - Community Validator",
         "longDescription": "CertHum is a community validator that provides Web3 infrastructure in support of decentralization.",
         "isOrganization": true
-     }
+     },
+    {
+        "address": "15vvAhHcmop8PjP5QNr3mXzX1oesv6vjun25iSgMe9YpDezv",
```
Comment (stepanLav) [inline registry/kusama.json:195]: For Kusama should be used address in Kusama format -https://wiki.polkadot.network/docs/learn-account-advanced#address-format
please, fix it and add proof of ownership

**Comment by stepanLav** [comment]
@robinhodl69 please, fix Kusama address and the we will be able to merge your request. This is how it will looks like in the app:
<details>
  <summary>Spoiler!</summary>

Screenshot 2023-06-23 at 08 08 49
Screenshot 2023-06-23 at 08 08 54


</details>

**Comment by stepanLav** [comment]
@robinhodl69 
I apologise, but if you don't resolve the issue in your pull request, I'll have to close it as no longer relevant. Please make the necessary changes.
Thanks!

**Pull request closed without merging**